scripts/auto_pr_babysitter.py

```
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+import json
+import subprocess
+import time
+
+REPO_OWNER = "organvm"
+REPO_NAME = "peer-audited--behavioral-blockchain"
+REPO = f"{REPO_OWNER}/{REPO_NAME}"
+ROOT_DIR = "/Users/4jp/Workspace/peer-audited--behavioral-blockchain"
+
+def run_cmd(cmd, cwd=ROOT_DIR):
+    res = subprocess.run(cmd, shell=True, capture_output=True, text=True, cwd=cwd)
```

Ruff (0.15.21)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)